tap-gui/tap-gui-rbac/view-resouces-rbac.md

```diff
@@ -0,0 +1,77 @@
+# View Runtime Resources on Authorization-Enabled Clusters in Tanzu Application Platform GUI
+
+To visualize runtime resources on Authorization-enabled clusters in Tanzu Application Platform GUI, you will need to proceed to the Software Catalog `Component` of choice and click on the `Runtime Resources` tab on the top of ribbon.
+
+![Screenshot of Runtime Resources](./../images/tap-gui-multiple-clusters.png)
+
+Once you click on `Runtime Resources`, the system shall prompt for your OIDC credentials. Tanzu Application Platform GUI will use your credentials to query the clusters for the respective Runtime Resources.
+
+Visibility of remote clusters that are not restricted by Authorization, is done through the general Service Account of Tanzu Application Platform GUI and is not restricted for users. For more information on how to set up unrestrictred remote cluster visibility, please refer to [Viewing resources on multiple clusters in Tanzu Application Platform GUI](./../cluster-view-setup.md).
+
+The type of query to the remote cluster will depend on the definition of the Software Catalog `Component`. In Tanzu Application Platform GUI, there are two types of `Components`:
+  
+  - [Globally-scoped Components](#-globally-scoped-components)
+  - [ Namespace-scoped Components](#-namespace-scoped-components)
+
+This property of the Component is important to distinguish because it is going to affect runtime resource visibility depending on your permissions on a specific cluster.
+
+If your permissions on the Authorization-enabled cluster are limited to specific namespaces, you will not have visibility into `Runtime Resources` of globally-scoped Components.
+
+To have visibility into `Runtime Resources` of globally-scoped Components, you will need to have cluster-scoped access.
+
+## <a id="globally-scoped-components"></a> Globally-scoped Components
+
+For globally-scoped Components, when you access `Runtime Resources`, Tanzu Application Platform GUI will query all Kubernetes namespaces for runtime resources that with a matching `kubernetes-label-selector` (usually with a `part-Of` prefix).
+
+For example, `demo-component-a` does not have a `kubernetes-label-selector` in the metadata.annotations section, making is a globally-scoped component:
```

Comment:
It should be `backstage.io/kubernetes-namespace` instead of `kubernetes-label-selector`

Reply:
Thanks, @dbravovmw, please raise a PR to change every bad instance of this. (`kubernetes-label-selector` appears many times in the topic.)

https://github.com/pivotal/docs-tap/blob/main/tap-gui/tap-gui-rbac/view-resouces-rbac.md

Reply:
I now realize @dbravovmw was likely talking about line 26 here only, and I've made that change on behalf of @andreizimin. @dbravovmw, please correct me if I'm wrong.

Reply:
Bump for @dbravovmw

Reply:
Yes, that is correct. Sorry for the late reply but I was on holidays :)